xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Visualization.h

```diff
@@ -656,6 +656,7 @@ namespace addon
       m_instanceData->toAddon->start = ADDON_Start;
       m_instanceData->toAddon->stop = ADDON_Stop;
       m_instanceData->toAddon->audio_data = ADDON_AudioData;
+      m_instanceData->toAddon->is_dirty = ADDON_IsDirty;
```

Comment:
Found here also a big bug, the address was never set therefore the function was never used.

I just noticed now. You should not have the debug view on the screen when testing :joy: (this then make his "IsDirty" always itself :smirk: and therefore seen only now).